[2024-09-25T07:47:04.143+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-25T07:47:04.167+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: random_forest_classifier_dag.rfc_save_results scheduled__2024-09-24T01:00:00+00:00 [queued]>
[2024-09-25T07:47:04.180+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: random_forest_classifier_dag.rfc_save_results scheduled__2024-09-24T01:00:00+00:00 [queued]>
[2024-09-25T07:47:04.183+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-09-25T07:47:04.202+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): rfc_save_results> on 2024-09-24 01:00:00+00:00
[2024-09-25T07:47:04.214+0000] {standard_task_runner.py:64} INFO - Started process 16620 to run task
[2024-09-25T07:47:04.217+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'random_forest_classifier_dag', 'rfc_save_results', 'scheduled__2024-09-24T01:00:00+00:00', '--job-id', '591', '--raw', '--subdir', 'DAGS_FOLDER/***/dags/.ipynb_checkpoints/hw1-checkpoint.py', '--cfg-path', '/tmp/tmpsegc41qm']
[2024-09-25T07:47:04.220+0000] {standard_task_runner.py:91} INFO - Job 591: Subtask rfc_save_results
[2024-09-25T07:47:04.280+0000] {task_command.py:426} INFO - Running <TaskInstance: random_forest_classifier_dag.rfc_save_results scheduled__2024-09-24T01:00:00+00:00 [running]> on host 82997fc6deb7
[2024-09-25T07:47:04.382+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='alexvlg34@gmail.com' AIRFLOW_CTX_DAG_OWNER='Alex Trigolos' AIRFLOW_CTX_DAG_ID='random_forest_classifier_dag' AIRFLOW_CTX_TASK_ID='rfc_save_results' AIRFLOW_CTX_EXECUTION_DATE='2024-09-24T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-24T01:00:00+00:00'
[2024-09-25T07:47:04.384+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-25T07:47:04.404+0000] {hw1-checkpoint.py:167} INFO - Save metrics...
[2024-09-25T07:47:04.544+0000] {base.py:84} INFO - Using connection ID 's3_connection' for task execution.
[2024-09-25T07:47:04.546+0000] {connection_wrapper.py:388} INFO - AWS Connection (conn_id='s3_connection', conn_type='aws') credentials retrieved from login and password.
[2024-09-25T07:47:05.225+0000] {hw1-checkpoint.py:194} INFO - DONE!!!
[2024-09-25T07:47:05.227+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-25T07:47:05.228+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-25T07:47:05.240+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=random_forest_classifier_dag, task_id=rfc_save_results, run_id=scheduled__2024-09-24T01:00:00+00:00, execution_date=20240924T010000, start_date=20240925T074704, end_date=20240925T074705
[2024-09-25T07:47:05.315+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-09-25T07:47:05.342+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-25T07:47:05.345+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
